### ISSUE TITLE
Rton-Json working fine, but json-rton not

### ISSUE THREAD

**Issue by wizcat28** [oobol2000oo/RETON#18]
I tried using a custom save file and yknow slamming  it through and then editing it and reprocessing back to rton, but it only sends the logs without the actual json-rton (github doesnt support json files so i cant send to you the edited one), you may have corrupted the key sorting a bit in your latest patch, im going to try and older version and see what happens

The changes i made to it  were the levels (l variable in the bottom section of the json) and amount of plants

**Comment by wizcat28** [comment]
i think i probably misunderstood how to use the program, any clarification would be nice :)

**Comment by h3x4n1um** [comment]
> 
> 
> i think i probably misunderstood how to use the program, any clarification would be nice :)

You can drag and drop the file into the exe and done 

Or you input path to file or folder you want to convert into the exe

**Comment by h3x4n1um** [comment]
> 
> 
> I tried using a custom save file and yknow slamming it through and then editing it and reprocessing back to rton, but it only sends the logs without the actual json-rton (github doesnt support json files so i cant send to you the edited one), you may have corrupted the key sorting a bit in your latest patch, im going to try and older version and see what happens
> 
> The changes i made to it were the levels (l variable in the bottom section of the json) and amount of plants

I think PopCap implemented some sort of anti cheat in `pp.dat`, if you change the `pp.dat` the game will reset all your progress

**Comment by nineteendo** [comment]
If you want to edit your pp.dat for online use. You could reset PVZ2 and then add your modified pp.dat.

**Comment by eris-webserv** [comment]
Does PVZ2 Eclise have this same protection? I want to edit in some gems.

**Comment by nineteendo** [comment]
Yes, it has the same "protection": whether the keys are in the correct order and the data format is correct.
You can try this converter: https://github.com/Nineteendo/PVZ2tools/releases/tag/1.1
